.travis.yml

```diff
@@ -91,8 +92,8 @@ script:
       python3 -m kodi_addon_checker --branch=$BRANCH ../repo;
     fi
   - if [[ -z "$REPO" || -z "$BRANCH" ]]; then
-      py.test -v;
+      pylint * --rcfile=.pylintrc;
     fi
   - if [[ -z "$REPO" || -z "$BRANCH" ]]; then
-      pycodestyle . --max-line-length=120;
+      py.test -v;
```

Comment:
What's the reason for running the code checker before the tests?

Reply:
kodi checker will create log files

Reply:
@Rechi  Should I run py.test before?